backend/src/models/person.model.ts

```diff
@@ -4,24 +4,30 @@
 import mongoose, { Schema, model } from 'mongoose';
 
 export interface PersonModel {
-  first_name: string,
-  last_name: string,
+  full_name: string,
   birthday: Date,
+  gender: string,
+  location: string,
+  date_met: Date,
+  how_we_met: string,
   interests: string[],
   organisation: string,
-  time_added: Date,
```

Comment:
How come time_added isn't a thing anymore? I think we need this field to be able to display things on the dashboard for frontend

Reply:
Got it, have added it back 👍